src/js/fc.js

```diff
@@ -692,6 +692,13 @@ var FC = {
     getPidDefaults: function() {
         var versionPidDefaults = DEFAULT_PIDS;
         // if defaults change they should go here
+        if (semver.gte(CONFIG.apiVersion, "1.43.0")) {
```

Comment:
You are changing `DEFAULT_PIDS` **after** you assign it to `versionPidDefaults` - this is definitely not the way it should be.
Another no-no is that this is a getter with side-effects. Please do this in another way.

Reply:
thanks, here is the problem yes, fixed now